cmd/utils.go

```diff
@@ -93,7 +93,14 @@ func path2BucketObject(s string) (bucket, prefix string) {
 }
 
 func getDefaultParityBlocks(drive int) int {
-	return drive / 2
+	switch drive {
+	case 4, 5:
+		return 2
+	case 6, 7:
+		return 3
+	default:
+		return 4
+	}
```

Comment:
this seems to be an important change that needs to be called out?

Reply:
i.e default parity drives for 16 drive erasure set is 4 now

Reply:
Correct this is the new change @krishnasrinivas, I have one pending change in this PR to ensure that we honor new values only for fresh deployment.

Reply:
There's also website copy + docs that needs to be updated in-sync with this.